.gitlab-ci.yml

```diff
@@ -121,12 +121,26 @@ gpu_pre_build:
         - echo $commit_author_name
     script:
         - echo "HOST arch.:" $HOST
-        - echo "GPU vendor:" $GPU_VENDOR ", GPU model:" $GPU_MODEL
+        - echo "GPU model:" $GPU_MODEL
         - nvidia-smi
 
 
 .common_gpu_test_script: &common_gpu_steps
     - export CTEST_OUTPUT_ON_FAILURE=1
+    - if [[ "${BACKEND}" == "hipsycl" ]]; then
+        export CC=/usr/bin/gcc-9 ;
+        export CXX=/usr/bin/g++-9 ;
+        export FC=/usr/bin/gfortran-9 ;
```

Comment:
Hi, 
yep we need 2 compilers because hipsycl v0.9.1 can be compiled only with gcc@9. But cuda@10 requires gcc@8. The problem is the nvidia driver on our GPU CI runner. In short, it needs to be updated. But because the runner doesn't fully belong to us I cannot do it. However, once it is done we can switch to cuda@11 and then we will need only gcc@9

You're correct. A comment is a good idea.

Reply:
done